### ISSUE TITLE
Ombi Auto Run User Importer No Longer Functional

### ISSUE THREAD

**Issue by Archmonger** [causefx/Organizr#946]
###### Ombi Version: 3.0.3477 
<hr>

##### Problem Description:
Ever since jumping from Beta 200 to Beta 400, I haven't had any luck getting the auto run user importer function to work. Based on Organizr Logs, it never even attempts to run.
<hr>

##### Reproduction Steps:
Invite a user and watch as Ombi does not run the user importer, and no organizr logs are created regarding anything related to the Ombi User Importer.
<hr>

**Comment by causefx** [comment]
are you saying the importer ran on beta 200?

**Comment by Archmonger** [comment]
Yeah, either that or beta 100. I watch your github push logs pretty closely (As all Groots do), and when ever one you first pushed the commit with the Ombi Auto User Importer, it worked perfectly. Every other Ombi related function in OrgV2 appears to work though. The times I've made pulls were on 100, 200, 400, and 600, but I can't recall if it was 200 or 400 where I first discovered it was broken.

**Comment by causefx** [comment]
i just tried and it worked:

![image](https://user-images.githubusercontent.com/16184466/46921844-126d8e00-cfb5-11e8-97ea-8ffb01452230.png)

I just added more logging for it, so if you want to force update and test again then check logs.

**Comment by Archmonger** [comment]
![image](https://user-images.githubusercontent.com/16909269/46998334-1a255380-d0d7-11e8-947b-239fd81b96dc.png)
Organizr logs indicate a successful request to ombi to run the user importer. The user was not imported though. Manually running the user importer afterwards works perfectly. I will update Ombi (currently 3.0.3477) and let you know if that fixes the issue.

**Comment by Archmonger** [comment]
Tested on 3.0.3795, non-functional auto import, but manual import works. Same behavior as 3.0.3477.

**Comment by causefx** [comment]
it seems the API changed from GET to POST.

adding fix now

**Comment by Archmonger** [comment]
API call works. I recommend adding an additional ~5 second wait time on first login to allow for Ombi's user importer to run. As it currently stands, if the user quickly enters the Ombi tab he/she will get stuck without SSO.

**Comment by causefx** [comment]
i had sso on registration in wrong place, i will submit fix to try now
